google/cloud/security/inventory/inventory_loader.py

```diff
@@ -194,15 +192,15 @@ def _send_email(organization_id, cycle_time, cycle_timestamp, status, pipelines,
 
     for pipeline in pipelines:
         try:
-            pipeline['count'] = dao.select_record_count(
-                pipeline['pipeline'].RESOURCE_NAME,
+            pipeline.count = dao.select_record_count(
+                pipeline.name,
                 cycle_timestamp)
         except MySQLError as e:
             LOGGER.error('Unable to retrieve record count for %s_%s:\n%s',
-                         pipeline['pipeline'].RESOURCE_NAME,
+                         pipeline.name,
                          cycle_timestamp,
                          e)
-            pipeline['count'] = 'N/A'
+            pipeline.count = 'N/A'
```

Comment:
If some new code expects pipeline.count to be an int() this will blow up. I think set to None to show there was an error and the result is unknown.